pkg/pipeline/provider.go

```diff
@@ -3,27 +3,26 @@ package pipeline
 import (
 	"context"
 	"fmt"
-	"net/http"
-	"net/url"
-	"regexp"
 
-	"github.com/go-resty/resty/v2"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/jfrog/terraform-provider-shared/client"
+	"github.com/jfrog/terraform-provider-shared/util"
 )
 
 var Version = "0.0.1"
+var productId = "terraform-provider-pipeline/" + Version
 
 func Provider() *schema.Provider {
 	p := &schema.Provider{
 		Schema: map[string]*schema.Schema{
 			"url": {
 				Type:         schema.TypeString,
 				Required:     true,
-				DefaultFunc:  schema.MultiEnvDefaultFunc([]string{"PIPELINES_URL", "JFROG_URL"}, "http://localhost:8081"),
+				DefaultFunc:  schema.MultiEnvDefaultFunc([]string{"PIPELINES_URL", "JFROG_URL"}, "http://localhost:8082"),
 				ValidateFunc: validation.IsURLWithHTTPorHTTPS,
```

Comment:
Please use new style validation

Reply:
I'm not planning to update the code in this PR.